ocaml-lsp-server/test/e2e-new/code_actions.ml

```diff
@@ -1288,25 +1288,10 @@ let f (x : t) (d : bool) =
     let end_ = Position.create ~line:5 ~character:5 in
     Range.create ~start ~end_
   in
-  print_code_actions source range ~filter:(find_action "merlin-jump");
+  print_code_actions source range ~filter:(find_action "merlin-jump-match");
```

Comment:
Now that only one precise jump is printed, could you expand the test to show that the jumps to let/fun/next-case are still there ?